tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -281,12 +281,27 @@ async def test_client_key_exchange_command(self, bft_network):
             all_client_ids=bft_network.all_client_ids()
             log.log_message(message_type=f"sending client key exchange command for clients {all_client_ids}")
             op = operator.Operator(bft_network.config, client, bft_network.builddir)
-            rep = await op.client_key_exchange(all_client_ids)
+            rep = await op.client_key_exchange_command(all_client_ids)
             rep = cmf_msgs.ReconfigurationResponse.deserialize(rep)[0]
             assert rep.success is True        
             log.log_message(message_type=f"block_id {rep.response.block_id}")
             assert rep.response.block_id == 1
     
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
+    async def test_client_exchange_public_key(self, bft_network):
```

Comment:
Currently, we are not validating the client reconfiguration commands (it will be done in the near future), once it is done, this test won't work as the operator key won't match the expected signature (unless we sign with the actual bft client id). Do u think we should enable this kind of operation from the operator as well?

Reply:
We'll change the tests